docs/Installing.md

```diff
@@ -0,0 +1,9 @@
+# Installing
+
+Do you have to use `[composer](https://getcomposer.org)` to install this library.
+
+```sh
+composer require roave/strict-php
+```
+
+**Next step:** [Enabling StrictPhp](bootstrap.md)
```

Comment:
Add notes about the fact that `strict-php` is only available with `"minimum-stability": "dev"` right now